configuration.json

```diff
@@ -45,13 +45,13 @@
     }
   },
   {
-    "key": "updateOnCorrelate",
+    "key": "updateOnUpdate",
```

Comment:
This key has been changed or added, but nowhere in the scripts has it been used. You have two options: either modify the scripts to check and utilize this key, or, if it’s not needed, remove it from the configuration.

Reply:
fixed, removed the keys